src/components/UserCard.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react'
+
+import axios from 'axios';
+
+export default class UserCard extends Component {
+
+    constructor(){
+        super();
+        this.state={
+            userData:{},
+            userName:"ravi771999",
+        }
+    }
+
+    async componentDidMount(){
+        let response=await axios.get(`https://api.github.com/users/${this.state.userName}`);
```

Comment:
create a common util for hitting the api and also handle the success and reject state in it. Read about Promises. Do at all places